db/migrate/20160115142023_remove_replicated_rows_from_newly_excluded_tables.rb

```diff
@@ -0,0 +1,25 @@
+class RemoveReplicatedRowsFromNewlyExcludedTables < ActiveRecord::Migration
+  class MiqEventDefinition < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ScanItem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Configuration < ActiveRecord::Base
+    serialize :settings, Hash
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    region_cond = ActiveRecord::Base.region_to_conditions(File.read(File.join(Rails.root, "REGION")).to_i)
+    MiqEventDefinition.where.not(region_cond).delete_all
+    ScanItem.where.not(region_cond).delete_all
```

Comment:
So it turns out that rubyrep will remove the triggers and clean up the pending_change and sync_state tables [here](https://github.com/ManageIQ/rubyrep/blob/v1.2.0-7/lib/rubyrep/replication_initializer.rb#L318), but it doesn't actually remove the rows from the target db.

So if we want to do that it looks like we'll have to do it ourselves.